video/out/cocoa_common.m

```diff
@@ -574,7 +574,6 @@ int vo_cocoa_config_window(struct vo *vo)
         if (!s->embedded && s->window) {
             if (reset_size)
                 queue_new_video_size(vo, width, height);
-            vo_cocoa_fullscreen(vo);
```

Comment:
yes you are right about this, it doesn't work anymore. think it's worth to only call it when "vo->opts->fullscreen" is true? that way we can avoid unnecessary calls.